packages/expo-crypto/ios/CryptoModule.swift

```diff
@@ -47,6 +49,20 @@ private func digestString(algorithm: DigestAlgorithm, str: String, options: Dige
   }
 }
 
+private func getRandomValues(array: TypedArray) throws -> TypedArray {
```

Comment:
According to the [specification](https://w3c.github.io/webcrypto/#Crypto-method-getRandomValues) we could consider throwing `QuotaExceededException` when `array.byteLength` is greater than 65536 🤔 I bet no one will try to do that, but let's try to be more compliant with the spec 😉 
It wouldn't map to JS `QuotaExceededError`, but we don't need to go that far 😅

Reply:
We decided to not do it for now